cpp/src/arrow/compute/kernels/vector_replace.cc

```diff
@@ -83,11 +87,19 @@ struct CopyArrayBitmap {
     BitUtil::SetBitTo(out_bitmap, out_offset,
                       BitUtil::GetBit(in_bitmap, in_offset + offset));
   }
+
+  void SetBitTrue(uint8_t* out_bitmap, int64_t out_offset, int64_t offset) const {
+    BitUtil::SetBitTo(out_bitmap, out_offset, true);
+  }
 };
 
 struct CopyScalarBitmap {
   const bool is_valid;
 
+  const uint8_t* GetInBitmap() const {
+    return nullptr;
+  }
+
```

Comment:
So maybe instead of creating this weird method, you can create a property like this:
```cpp
  const static uint8_t* in_bitmap = nullptr;
```

Reply:
solved